Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -6,6 +6,7 @@ Authors: Johannes Hölzl, Mario Carneiro, Floris van Doorn
 import Mathlib.Data.Fintype.BigOperators
 import Mathlib.Data.Set.Countable
 import Mathlib.Logic.Small.Set
+import Mathlib.Logic.UnivLE
```

Comment:
This import is required for the `Small.{max u v, u}` instance. This should be documented somewhere...

Reply:
Documented how? Like any import, presumably removing it breaks the file.

Reply:
Well, the module docstring for `Logic.UnivLE` says nothing about how the `UnivLE` API indirectly provides this instance. I'll make a PR to add this statement.